config/locales/views/en.yml

```diff
@@ -410,6 +410,10 @@ en:
         restricted_html: Inspired by [Restricted Work] by %{creator_link}
         revealed_html: Inspired by %{work_link} by %{creator_link}
         unrevealed: Inspired by a work in an unrevealed collection
+      translated_to:
+        restricted_html: 'Translation into %{language} available: [Restricted Work] by %{creator_link}'
+        revealed_html: 'Translation into %{language} available: %{work_link} by %{creator_link}'
+        unrevealed_html: 'Translation into %{language} available: A work in an unrevealed collection.'
```

Comment:
The spreadsheet of new copy has a full-stop here, but none of the others do. Is that intentional, or should we remove it for consistency?

Reply:
Remove, please!